### DISCUSSION TITLE
Github Action Notification Preferences

### DISCUSSION THREAD

**Discussion by steveoh** [community/community#4369] [Actions]
It would be great to have a few more knobs and buttons for github action notifications. 

We created a replacement for the probot reminder app with two github actions that read every issue comment and then scan issue comments on a schedule. 

This fills up our notifications pretty quickly. 

I think great improvement would be gained by providing the ability to mute successful notifications or mute the entire action run notifications.

**Comment by rajbos** [comment]
And to add to that: If I have the workflow open during execution, I really do not need to get a notification that the workflow fails! (I am already look at it :-D ).
